src/components/ActivitiesListFilter.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const ActivitiesListFilter = () => (
+  <div>
+    <input type="text" />
```

Comment:
assumption was a little different. 
This input should trigger `onChange` event passed here as a prop. 
Error message also should be here and should be rendered when Input will get error from his parent, or/and on his own blur. So -> onBlur event should be handle here.

Reply:
sorry, I've missed something. This comment is not valid.

Reply:
why not reuse `Input` component here?